app/views/devise/registrations/edit.html.erb

```diff
@@ -3,40 +3,43 @@
 <%= form_for(resource, as: resource_name, url: registration_path(resource_name), html: { method: :put }) do |f| %>
   <%= bootstrap_devise_error_messages! %>
 
-  <div class="form-group">
+  <div class="form-group mb-3">
     <%= f.label :name %>
     <%= f.text_field :name, autofocus: true, class: 'form-control' %>
   </div>
 
-  <div class="form-group">
+  <div class="form-group mb-3">
     <%= f.label :email %>
     <%= f.email_field :email, autocomplete: 'email', class: 'form-control' %>
   </div>
 
-  <div class="form-group">
+  <div class="form-group mb-3">
     <%= f.label :password %>
     <%= f.password_field :password, autocomplete: 'new-password', class: 'form-control' %>
 
     <small class="form-text text-muted"><%= t('.leave_blank_if_you_don_t_want_to_change_it') %></small>
   </div>
 
-  <div class="form-group">
+  <div class="form-group mb-3">
     <%= f.label :password_confirmation %>
     <%= f.password_field :password_confirmation, autocomplete: 'new-password', class: 'form-control'  %>
   </div>
 
-  <div class="form-group">
+  <div class="form-group mb-3">
     <%= f.label :current_password %>
     <%= f.password_field :current_password, autocomplete: 'current-password', class: 'form-control' %>
 
     <small class="form-text text-muted"><%= t('.we_need_your_current_password_to_confirm_your_changes') %></small>
   </div>
 
-  <div class="form-group">
+  <div class="form-group mb-3">
+    <%= f.label :marketing_opt_in %>
+    <%= f.check_box :marketing_opt_in %>
+  </div>
+
+  <div class="form-group mb-3">
     <%= f.submit t('.update'), class: 'btn btn-primary' %>
   </div>
 <% end %>
 
-<p><%= link_to t('.cancel_my_account'), registration_path(resource_name), data: { confirm: t('.are_you_sure') }, method: :delete %>.</p>
-
-<%= link_to t('.back'), :back %>
```

Comment:
Is there a reason to remove this back link?

Reply:
I can't remember if there was a reason for removing it, happy to restore

Reply:
Restored!